src/storage-controller/src/lib.rs

```diff
@@ -1915,11 +1915,6 @@ where
                         .read_hold
                         .try_downgrade(Antichain::new())
                         .expect("must be possible");
-                } else {
-                    soft_panic_or_log!(
-                        "DroppedId for ID {id} but we have neither ingestion nor export \
-                         under that ID"
-                    );
```

Comment:
I think that soft panic should stay? Since we're doing the `active_copies` tracking there isn't a reason we should ever get here, right?